src/style.css

```diff
@@ -1 +1,52 @@
 /* styles go here */
+body {
+  margin: 8px 10px;
+}
+
+.form-field {
+  margin: 10px 0;
+}
+
+.fieldset {
+  margin: 20px 0;
+}
+
+.additional-info {
```

Comment:
Please remove all clases below. Just replace from margin: 10px 0; to margin-bottom: 10px; for .form-field selector.
Also dont forget remove this classes from all tags

Reply:
Did I understand correctly ?
I remove the lines from css: body, fieldset, additional-info and in the form-field I make a change
if i remove the fieldset line then i will not have indent between inputs 20  #px